### DISCUSSION TITLE
Settings used in the youtube video?

### DISCUSSION THREAD

**Discussion by SF73** [CodeFHD/OptiCore#6] [Q&A] [answered]
Hi again,

I guess that you're the author of this video : https://www.youtube.com/watch?v=D8rQBVI4lIg
May I ask what were your settings to get such results (volume scattering in the whole world and "quick" render time)

Thanks

**Comment by CodeFHD** [comment, marked as answer]
Hi,

attached below is the .blend-file that I used for the video. You should be able to see the setting therein.

[opticoredemo.zip](https://github.com/CodeFHD/OptiCore/files/6423828/opticoredemo.zip)

In case of issues, please note that it was created with Blender 2.83 and LuxCore v2.4 or some 2.5 alpha (i don't remember exactly but it shouldn't matter)

the most important aspects that I can think of spontaneously:
- I used "Light tracing" here, but "BiDir" could work, too.
- For Volumes, the "scattering scale" paramter depends on the size of your scene. I usually work with assumed units of millimeters, even thoug technically "blender units" are meters.
- Example: with a lens of 10 "Blender-units" diameter, I use a scattering scale of around 0.01.
- With LuxCore, light tracing isn't visible on the first bounce. I use a laser light source, but directly in front of it I have a thin glass panel that it passes through, so that the scattering you see isn't the first bounce.

I hope this helps. if not, feel free to discuss further.
